example/advanced/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  bitcoind:
+    image: zquestz/bitcoin-abc
+    # specify additional command line arguments here
+    command: bitcoind
+    healthcheck:
+      test: ["CMD", "/entrypoint.sh", "bitcoin-cli", "getinfo"]
+    ports:
```

Comment:
Pretty sure you need to expose the zeromq port here. That way other containers can reach it.